optuna/trial/_trial.py

```diff
@@ -396,6 +400,8 @@ def objective(trial):
 
         Returns:
             A suggested value.
+        .. seealso::
```

Comment:
```suggestion

        .. seealso::
```

Reply:
I forgot to insert a blank line. Thanks for the kind remarks. I reflected your comment to the code.

Reply:
Thanks!